api/src/main/java/org/apache/cloudstack/vm/UnmanagedInstance.java

```diff
@@ -0,0 +1,333 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.cloudstack.vm;
+
+import java.util.List;
+
+public class UnmanagedInstance {
```

Comment:
Can you please suggest a rename?
This is just a convenience class to pass data retrieved from VMware(or other hypervisors) to managementserver. Since nothing is stored in DB I didn't used `VO` in name. There is a different UnmanagedInstanceResponse class added for API. As far as I know we use TO when we pass objects to agents.

Reply:
A TO would be apropriate, than.

Reply:
Renamed